src/services/dashboards/dashboard-detail/modules/DashboardWidgetContainer.vue

```diff
@@ -174,14 +183,13 @@ export default defineComponent<Props>({
             state.widgetRef.forEach((d:any) => {
                 if (typeof d?.refreshWidget === 'function' && state.initiatedWidgetMap[d.$el.id]) promises.push(d?.refreshWidget());
             });
-            emit('update:loading', true);
             await Promise.allSettled(promises);
-            emit('update:loading', false);
         };
         expose({
             refreshAllWidget,
         });
-        onMounted(() => {
+        onMounted(async () => {
+            await store.dispatch('reference/loadAll');
```

Comment:
question: DashboardDetailPage has `reference/loadAll` already. Should I delete it?

Reply:
Oh, I already removed it!

Reply:
Oh I missed it😅 Thanks!